lib/features/search/mailbox/presentation/search_mailbox_controller.dart

```diff
@@ -898,6 +900,13 @@ class SearchMailboxController extends BaseMailboxController with MailboxActionHa
     FocusManager.instance.primaryFocus?.unfocus();
   }
 
+  void autoCreateVirtualFolder() {
+    addFavoriteFolderToMailboxList();
+    if (dashboardController.isAINeedsActionEnabled) {
+      addActionRequiredFolder();
+    }
+  }
```

Comment:
Duplicated with method in mailbox_controller, can we remove it

Reply:
Combine to one method `autoCreateVirtualFolder` to base_mailbox_controller